components/script/dom/testbinding.rs

```diff
@@ -428,6 +429,9 @@ impl TestBindingMethods for TestBinding {
     fn PassByteString(&self, _: ByteString) {}
     fn PassEnum(&self, _: TestEnum) {}
     fn PassInterface(&self, _: &Blob) {}
+    fn PassTypedArray(&self, _: CustomAutoRooterGuard<typedarray::Int8Array>) {}
+    fn PassTypedArray2(&self, _: CustomAutoRooterGuard<typedarray::ArrayBuffer>) {}
+    fn PassTypedArray3(&self, _: CustomAutoRooterGuard<typedarray::ArrayBufferView>) {}
```

Comment:
It might be nice to typedef these into `Int8Array`, etc.

Reply:
How should we deal with nullable `CustomAutoRooterGuard<Option<Typedarray::Int8Array>>` types then? Should we simply pass only reference to rooted arguments, hiding the `CustomAutoRooterGuard` 'implementation detail'?

Reply:
Hmm. Ideally we would have `Option<CustomAutoRooterGuard<Typedarray::Int8Array>>` instead. Possibly that just requires rearranging the code that applies the CustomAutoRooterGuard templated type?

Reply:
Although really passing `&typedarray::Int8Array` sounds really nice too.